resources/views/dashboard/frontend/index.blade.php

```diff
@@ -81,6 +81,37 @@ class="block-content block-content-full d-flex align-items-center justify-conten
                     </a>
                 </div>
             </div>
+            <div class="row">
+                <div class="col-md-6 col-xl-6 col-lg-6 col-12 col-xs-12 invisible" data-toggle="appear">
+                   <a class="
+                            block
+                            block-transparent
+                            block-link-pop
+                            w-100
+                            h-300
+                            d-md-flex
+                            align-items-md-stretch
+                            bg-image"
+                        style="
+                        background-image : url(https://www.bravissamenvitaal.nl/wp-content/uploads/2020/02/iStock-1058457940-2-495x400.jpg); 
```

Comment:
De image mag je gewoon in storage knallen hoor en dan van uit storage ophalen, wat als die image ooit verwijderd word?

Reply:
Also moest de rest ook niet een afbeelding krijgen?